src/OpenRiaServices.Tools/Test/Utilities/TestInitializer.cs

```diff
@@ -37,19 +45,19 @@ public static void AssemblyInit(TestContext context)
         }
 
 
+#if NETFRAMEWORK
         private static void RegisterMSBuildAssemblyResolve()
         {
             var vsInstance =
-            Microsoft.Build.Locator.MSBuildLocator.QueryVisualStudioInstances()
+            Microsoft.Build.Locator.MSBuildLocator.QueryVisualStudioInstances(new Microsoft.Build.Locator.VisualStudioInstanceQueryOptions()
```

Comment:
low prio: this code can be changed back to just Microsoft.Build.Locator.MSBuildLocator.QueryVisualStudioInstances()